CHANGES.rst

```diff
@@ -367,6 +369,8 @@ New Features
 
 - ``astropy.vo``
 
+  - KeyError when converting Table v1.2 numeric arrays fixed. [#4781]
```

Comment:
@al-niessner , it's all part of the fun of open-source project contribution. :smile: